lib/global.js

```diff
@@ -0,0 +1,82 @@
+// Include Gulp, plugins and local files
```

Comment:
`global.js` should be renamed as `utils.js` and placed within the `tasks` folder